docs/code_contribution_guidelines.md

```diff
@@ -269,6 +270,33 @@ a good thing.
   wrap our plain text emails such that there’s room for a few levels of nested
   reply indicators without overflow in an 80 column terminal.
 
+<a name="HandlingModuleBreakingChanges" />
+
+### 4.5 Handling Module Breaking Changes
+Pull requests that introduce breaking changes to modules must ensure dependent 
+modules on the change are updated to referrence newly bumped module versions.  
+The purpose of this is to improve the release process by handling version 
+updates with the pull request that introduces it instead of waiting till 
+release.
+
+The request submitter should ensure the following steps are covered whenever a 
+new breaking change to a module's API is introduced:
+
+- Bump the major version in the `go.mod` of the affected module if not already 
+  done since the last release tag. Note that v1 modules are stablilized even 
+  without a release, breaking changes to such modules will require a major 
+  version bump as well.
+
```

Comment:
There shouldn't be blank lines between the bullets or it will make the formatting inconsistent.